### ISSUE TITLE
WCAG 3 Review: Terminology, structure, and style - suggested improvements

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#610]
Related to the question posed in the introduction:
>Are there usability improvements that would make WCAG 3 easier to use and find information? Is the structuring of the How-To, outcomes, and methods clear? Are there improvements to structure and style that you would like to see? Is there another way we can make content more usable?

**Answer:** As for usability, the terminology is still tripping our team up. We also have thoughts on organization/structuring that you might consider.

- How-to vs. method: How-to and method are difficult to parse what is the difference. It's so different from WCAG 2.x and the structure of finding guidance that When “How-to” is looked up in the dictionary, one of the definitions is “a practical method or instruction”. For those new to accessibility and people already familiar with WCAG 2.x, this makes it very difficult to remember which one to reference when wanting code examples (what were techniques in WCAG 2.x) and which is just an explanation (understanding in WCAG 2.x). I kept finding myself navigating to both the"outcomes" and the How-to looking for the WCAG 2.x guidance called "techniques". How-to as a term is a good meaningful term for the non-normative instructional information, but perhaps the methods or techniques should be included in the How-to content rather than a separate document. One-stop shopping for all guidance regarding a requirement.
- Guideline in WCAG 2.x vs Guideline in WCAG 3.0: It's confusing that guidelines in the two standards seem to be defined differently. In WCAG 2.1, it seems guidelines are non-normative, as it states: 
    >"Guidelines - Under the principles are guidelines. The 13 guidelines provide the basic goals that authors should work toward in order to make content more accessible to users with different disabilities. The guidelines are not testable, but provide the framework and overall objectives to help authors understand the success criteria and better implement the techniques.

    Where in WCAG 3.0 Guidelines section it states:
    >This section (with its subsections) provides requirements which must be followed to conform to the specification, meaning it is [normative](https://www.w3.org/TR/wcag-3.0/#dfn-normative).

    It leads the reader to believe guidelines (with the embedded outcomes) are normative. However, if the guidelines are not in and of themselves testable, should they be normative? It doesn't seem so. The definitions and explanations of "Guidelines" should align better with WCAG 2.x, as the concept for guidelines hasn't really changed in WCAG 3.0.
- success criteria vs. “Outcome” - Use of "outcome" rather than "success criteria" may seem like a small change, but unfortunately there's a long history in WCAG of using "success criteria" as a term and all of these terminology changes make it more difficult to easily pick up and run with the new standard. Though "outcome" is a usability term for an intended consequence, it does not imply “required” where the "criteria" part of SC does. IMO the term “requirement” or "criterion" is more understandable than “outcome” as something normative. Otherwise the definition and uses of "outcome" need to be very specific that these are the normative parts.

More thoughts on organization/terminology:
- **How-to's:**
    - wondering if “Understand” may be better than “Get Started” for the first tab name. This will be a more similar term to WCAG 2.x, so an easier leap in terminology.
    - When there’s technology-specific information in the getting started tab, it needs to be identified as such. That way a reader can easily identify where the examples may differ in their technology (or ignore them). An example is in Captions where the first few sentences in the How-to's getting started are talking about XR requirements for captions. If Methods document is kept, there should be some cross link from How-tos to the methods. Right now you only get that link from the main WCAG 3.0 Guidelines doc. 
    - Thinking out of the box… The examples in the how-to could be what links to the methods or there could be another tab with Methods to cut down on the number of documents to link to. Less chance of getting lost in the many resources.
- **Left navigation:** There are no numbers associated with each outcome and none are listed in the left navigation. Wondering if this will make it harder to find specific outcomes if you don’t remember the exact short name. Time will tell as this gets more developed.
- **Outcomes and methods** pages (e.g. [Outcome: Text alternative available](https://www.w3.org/WAI/GL/WCAG3/2021/outcomes/text-alternative-available)
    -  the destination page lists “WCAG 3 How-Tos” at the top of the left navigation. This is confusing, as the links don’t take you to the How-to’s. 
    - The links in the left navigation are broken, so it is difficult to know what was intended destination. By the address, it seems it was supposed to go to outcomes pages.
- **Rating vs. scoring**
    - Both terms are used. Suggest using one to reduce confusion.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
This is ok to close.
